### DISCUSSION TITLE
The function of environment variable POSTGRES_DB

### DISCUSSION THREAD

**Discussion by hadmut** [docker-library/postgres#1148] [General]
Hi, 

I am not sure, whether this is a bug or intended. Documentation at https://hub.docker.com/_/postgres says

> POSTGRES_DB
> 
> This optional environment variable can be used to define a different name for the default database that is created when the image is first started. If it is not specified, then the value of POSTGRES_USER will be used.

But when using it, e.g. 

`podman run --rm -it -e POSTGRES_DB=xyz -e POSTGRES_PASSWORD=abc postgres:15-alpine`

it creates two databases,  postgres **and** xyz.  Based on the description, I would have expected xyz only. 

Is this intended or necessary behavior?

regards

**Comment by LaurentGoderre** [comment]
Running `\l+` in PSQL gives a description

```
  Name    |  Owner   | Encoding |  Collate   |   Ctype    | ICU Locale | Locale Provider |   Access privileges   |  Size   | Tablespace |                Description                 
-----------+----------+----------+------------+------------+------------+-----------------+-----------------------+---------+------------+--------------------------------------------
 postgres  | postgres | UTF8     | en_US.utf8 | en_US.utf8 |            | libc            |                       | 7453 kB | pg_default | default administrative connection database
 template0 | postgres | UTF8     | en_US.utf8 | en_US.utf8 |            | libc            | =c/postgres          +| 7297 kB | pg_default | unmodifiable empty database
           |          |          |            |            |            |                 | postgres=CTc/postgres |         |            | 
 template1 | postgres | UTF8     | en_US.utf8 | en_US.utf8 |            | libc            | =c/postgres          +| 7525 kB | pg_default | default template for new databases
           |          |          |            |            |            |                 | postgres=CTc/postgres |         |            | 
 xyz       | postgres | UTF8     | en_US.utf8 | en_US.utf8 |            | libc            |                       | 7297 kB | pg_default | 
```

**Comment by hadmut** [comment]
yeah, sure, but it still doesn't match the description of the docker environments. And „default administrative connection database“ can be just anything, it just says that someone thought there should be some database for whatever. 

And if it is described as „default“, it means „default“, and not „compulsory“. So what is the default needed for, if there is another database? Makes sense if no POSTGRES_DB is given. 

Just to clarify, if this is intended (and the documentation is misleading), or if this is an unintended side effect.
